populus/api/compile.py

```diff
@@ -0,0 +1,38 @@
+from .config import (
+    load_user_config
+)
+
+
+from populus.project import (
+    Project,
+)
+
+
+from populus.compilation.compile import (
+     compile_dirs,
+)
+
+from populus.compilation.helpers import (
+    write_compiled_sources,
+)
+
+
+def compile(project_root_dir, user_config_path=None):
```

Comment:
The word `compile` is a built-in and it's generally bad mojo to override those since it can be confusing for future consumers of this source code.

Can you rename this to something like `compile_project`?

Reply:
agreed I changed it several times, but this should not be tied to a project either